soroban-env-host/src/test/util.rs

```diff
@@ -145,4 +134,39 @@ impl Host {
     pub(crate) fn test_bin_obj(&self, vals: &[u8]) -> Result<HostObj, HostError> {
         self.to_host_obj(&self.test_bin_scobj(vals)?)
     }
+
+    pub(crate) fn raw_val_vec_to_sc_vec(&self, v: Vec<RawVal>) -> ScVec {
+        let mut res = Vec::<ScVal>::new();
+        for val in v {
+            res.push(val.try_into_val(self).unwrap());
+        }
+        res.try_into().unwrap()
+    }
+
+    // Registers a contract with provided WASM source and returns the registered
+    // contract ID.
+    // This relies on the host to have no footprint enforcement.
+    pub(crate) fn register_test_contract_wasm(
```

Comment:
I see that this was moved from host.rs, but I don't see `register_test_contract_token` anywhere. I'm guessing it was removed but not added back in because only the sdk uses it? Or do you have a pending sdk change that no longer requires it?

Reply:
I'll mark this resolved for now to unlblock the non-test changes. We can come back to this discussion on the respective SDK PR.